packages/logger/src/Logger.ts

```diff
@@ -1378,6 +1378,14 @@ class Logger extends Utility implements LoggerInterface {
       this.#bufferConfig.bufferAtVerbosity =
         LogLevelThreshold[bufferAtLogLevel];
     }
+    const AlcLogLevel =
+      this.getEnvVarsService().getAwsLogLevel() as keyof typeof LogLevelThreshold;
```

Comment:
`getAwsLogLevel` returns the string of the `AWS_LAMBDA_LOG_LEVEL` env var. I needed to cast it to a LogLevelThreshold enum. I considered whether this logic should be part of the `getAwsLogLevel` function.

Reply:
Agree - I'm inclined to suggest a light refactor.

I think we should:
- add a private `#alcLogLevel` prop to the class that starts as `undefined`
- during the `setOptions()` method, read the value with `this.getEnvVarsService().getAwsLogLevel()` and check it with `this.isValidLogLevel()`
- if valid, update `#alcLogLevel`, if not leave it as is

Then use that `#alcLogLevel` here, as well as in `awsLogLevelShortCircuit()` (also to refactor).

What do you think?

Reply:
Makes sense. Will do